subprojects/common/analysis/src/main/java/hu/bme/mit/theta/analysis/ptr/PtrUtils.kt

```diff
@@ -208,6 +210,9 @@ fun <S : ExprState> S.patch(writeTriples: WriteTriples): S =
   when (this) {
```

Comment:
Could we add these to a sealed ( :seal: ) hierarchy? Not necessarily as part of this PR, but more generally - I'm afraid we're missing places where the states/precisions/etc are not completely handled.